client/src/modules/profiles/component/ProfileForm.tsx

```diff
@@ -83,6 +92,23 @@ export const ProfileForm: React.FC<ProfileFormProps> = (props) => {
           />
         ),
       )}
+      <FormControl>
+        <Flex>
+          <FormLabel htmlFor="auto_subscribe">
+            Automatic chapter subscription
+          </FormLabel>
+          <Switch
+            id="auto_subscribe"
+            {...register('auto_subscribe')}
+            isDisabled={loading}
+          />
+        </Flex>
+        <FormHelperText>
+          {hasAutoSubscribe
+            ? 'Automatically subscribe to chapter when joining it and receive emails about new events in chapter.'
+            : "Don't automatically subscribe to joined chapters."}
```

Comment:
```suggestion
            ? '(After joining a new chapter, you will be notified when new events are created.)'
            : '(After joining a new chapter, you will not be notified about new events unless you subscribe.)'}
```
This way the information is describing the current state, rather than what will be the case once you click the button.  I believe this is the most common pattern